### ISSUE TITLE
directive is not working on abstract class fields

### ISSUE THREAD

**Issue by Diluka** [nestjs/graphql#766]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

directive is **not** working on abstract class fields

## Expected behavior

directive is working on abstract class fields

## Minimal reproduction of the problem with instructions

```ts
@ObjectType({ isAbstract: true })
export abstract class A { 
  @Directive('@upper')
  @Field()
  notWorking:string
}

@ObjectType()
export class B extends A {
  @Directive('@upper')
  @Field()
  working:string
}
```

https://codesandbox.io/s/nestjs-graphql-issue-766-lj8e8
## Environment

<pre><code>
Nest version: 7.2.1

 
For Tooling issues:
- Node version: 12  
- Platform:  

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Please provide a minimum reproduction repository.

**Comment by Diluka** [comment]
@kamilmysliwiec updated https://codesandbox.io/s/affectionate-goodall-lj8e8?file=/src/a.entity.ts

**Comment by kamilmysliwiec** [comment]
Fixed in 7.2.2
